src/core/webidl.js

```diff
@@ -234,7 +234,7 @@ function typeIsWhitespace(webIdlType) {
 }
 
 const extenedAttributesLinks = new Map([
-  ["CEReactions", "HTML51#cereactions"],
+  ["CEReactions", "!custom-elements#cereactions"],
```

Comment:
Looking over that it seems like HTMLConstructor is also pointing to the wrong place.